src/operation/decision_model.jl

```diff
@@ -480,7 +488,7 @@ function _write_model_dual_results!(
             key,
             timestamp,
             constraint,
-            [encode_key(key)],  # TODO DT: this doesn't seem right
+            [encode_key(key)],  # TODO DT: is this what the columns should be?
```

Comment:
This code is only called in the Simulation execution path. The original code used the name (Symbol) of the dual. Is this going to be correct?

Reply:
It shouldn't be needed anymore since the duals are not related to the constraints that have their own column names. I think we need to test this in an emulation test that asks for duals.

Reply:
If you're saying that that might expose a problem in the EmulationModel/InMemoryModelStore code, I agree.